index.html

```diff
@@ -131,21 +146,42 @@
       </section>
     </section>
 
-    <section class="appendix">
-      <h2>Issues</h2>
-
-      <p>The following issues are tracking aspects of the payment method identifier specification:</p>
-
-      <div class="issue" title="Write-up initial proposal for payment app registration spec">
-        <p>A registration mechanism may exist to install the code for new payment methods into the user agent. This process
-          would add support for a new payment method to the user agent. This mechanism should be defined in a separate specification.</p>
-        <p>There is an initial <a href="https://github.com/WICG/paymentrequest/blob/gh-pages/docs/registration.md">outline making a proposal</a>.</p>
-      </div>
-
-      <div class="issue" data-number="3" title="Should a payment method identifier (URL) resolve to a machine readable resource that describes it?">
-        <p>There is an open issue about whether payment method identifiers should resolve to a resource if they are URLs.</p>
-      </div>
+    <section>
+      <h2>W3C-Defined Strings as Payment Method Identifiers</h2>
+      <p>W3C expects to publish a small number of payment method
+	specifications that define abstractions across
+	similar payment methods (such as credit card payments,
+	tokenized payments, or credit transfers). W3C expects these
+	payment methods will be implemented in a wide variety of third party
+	payment apps. W3C identifies these payment methods with (short) strings; W3C expects to mint these strings in payment method specifications published by the Web Payments Working Group.</p>
+      <section>
+        <h2>Syntax</h2>
+	<p>The syntax of W3C strings used as payment method identifiers is constrained by this regular expression: <code>[a-z0-9-]+</code></p>
+      </section>
+      <section>
+        <h2>Matching</h2>
+        <p>For W3C strings, user agents MUST use exact string matching.</p>
+      </section>
     </section>
+	
+    <section>
+      <h2>Conditional Matching of Payment Methods</h2>
+      <p>For some payment methods, merchants may wish to express
+	that they accept a payment method but only under certain
+	conditions (e.g., "I only credit cards from brand A and
+	debit cards from brand B"). The Web Payments Working
+	Group has discussed several ways to support more sophisticated
+	matching, including:</p>
 
+      <ul>
+	<li>Parties may mint payment method identifiers to
+	  represent different conditions. This scalable approach may work
+	  well for a small number of independent conditions, but
+	  may not scale well for complex relationships.</li>
+	<li>Payment methods may define "filters." Merchants provide
```

Comment:
Your original suggestion had two parts: (1) filtering is defined in payment method specs. I think that is covered by the existing text. (2) support is not a normative requirement of this spec. I think silence on the matter is equivalent in this case. 

Your last comment is to say "why" there is no filtering behavior defined here. Do you have suggested text for that rationale?

Ian